objects/Citations/Citations.bib

```diff
@@ -65,3 +65,15 @@ @inproceedings{rompel1990one
   howpublished="\url{https://www.cs.princeton.edu/courses/archive/spr08/cos598D/Rompel.pdf}"
 }
 
+@article{luby1988construct,
+  author = {Luby, Michael and Rackoff, Charles},
+  title = {How to Construct Pseudorandom Permutations from Pseudorandom Functions},
+  journal = {SIAM Journal on Computing},
+  volume = {17},
+  number = {2},
+  pages = {373--386},
+  year = {1988},
+  doi = {10.1137/0217022},
+  publisher={SIAM},
+  howpublished="\url{https://pdfs.semanticscholar.org/7d58/9564998287a942149c4fee931852b22d8997.pdf}"
```

Comment:
This URL leads to a paper by Pieprzyk.

The paper you're referencing can be found at [https://epubs.siam.org/doi/abs/10.1137/0217022](https://epubs.siam.org/doi/abs/10.1137/0217022)